superset-frontend/plugins/plugin-chart-period-over-period-kpi/src/PopKPI.tsx

```diff
@@ -77,18 +135,26 @@ export default function PopKPI(props: PopKPIProps) {
             display: table-row;
           `}
         >
-          <ComparisonValue subheaderFontSize={subheaderFontSize}>
-            {' '}
-            #: {prevNumber}
-          </ComparisonValue>
-          <ComparisonValue subheaderFontSize={subheaderFontSize}>
-            {' '}
-            Δ: {valueDifference}
-          </ComparisonValue>
-          <ComparisonValue subheaderFontSize={subheaderFontSize}>
-            {' '}
-            %: {percentDifference}
-          </ComparisonValue>
+          {SYMBOLS.map((symbol, index) => (
+            <ComparisonValue
+              key={`comparison-symbol-${symbol}`}
+              subheaderFontSize={subheaderFontSize}
+            >
+              <SymbolWrapper
+                percentDifferenceNumber={
+                  index > 0 ? percentDifferenceNumber : 0
+                }
+                comparisonColorEnabled={comparisonColorEnabled}
+              >
+                {symbol}
+              </SymbolWrapper>
+              {index === 0
+                ? prevNumber
+                : index === 1
+                ? valueDifference
+                : percentDifferenceFormattedString}
```

Comment:
Hmm I'm not sure about coupling the values with indexes like this. I think it would be cleaner if we had some key-value mapping between symbols and values, or an array like [['#', prevNumber], ['△', valueDifference], ['%', percentDifferenceFormattedString]]